js/apps/admin-ui/src/components/json-file-upload/FileUploadForm.tsx

```diff
@@ -16,7 +18,6 @@ import {
 } from "react";
 import { DropEvent } from "react-dropzone";
```

Comment:
Looking at the PatternFly code it seems this type [already exists](https://github.com/patternfly/patternfly-react/blob/4ce551e98b489e4227306ed882e58d896e2fe810/packages/react-core/src/helpers/typeUtils.ts#L18) there, so we can likely remove the direct dependency on `react-dropzone` and use that type instead.